data/navigation.yaml

```
@@ -485,15 +485,25 @@ navbar-right:
   items:
   - <<: *digital_edition
   - <<: *portfolio
-  - <<: *my_account
+  - <<: *my_ft
 
+navbar-top-right:
+  label: Navigation
+  items:
+  - <<: *my_account
+  - <<: *header_subscribe
 
-navbar-right-anon:
+# To make the data better reflect the o-header markup
+# We are introducing a new navbar-top-right-anon object
+# As the navbar-right object is used for the right bottom row
+# For backwards compatibility we have kept navbar-right-anon as well
+navbar-top-right-anon: &navbar-top-right-anon
```

I don't know yaml.

I don't know yaml.

I don't know yaml.

But I think the `&` is setting an alias which I assume we don't need, because the name is the same?
```suggestion
navbar-top-right-anon:
```

---

I ALSO don't know yaml.

I ALSO don't know yaml.

I ALSO don't know yaml.


_Goes to check a few things_

---

Nope unfortunately it does appear to be needed. With your suggestion you get the following error `unidentified alias "navbar-top-right-anon"`

---

Thanks for checking – what a lazy reviewer I am 😇